src/components/common/topnav/TopNav.css.ts

```diff
@@ -15,6 +15,10 @@ export const TopNavContainer = styled(Nav)`
 	padding: 2% 10%;
 `;
 
+export const NavWrapper = styled.nav`
+	z-index: 50;
```

Comment:
Why was the z-index needed here?

Reply:
I will double check if it is still needed, but, originally it was because the nav inside the Logo (top left, topNav) was not clickable. I believe it was tucked behind the image. I tried to implement a few fixes so maybe the `z-index` isn't needed anymore.